drivers/net/loopback.c

```diff
@@ -89,6 +89,10 @@ static int loopback_send(const struct device *dev, struct net_pkt *pkt)
 
 	ARG_UNUSED(dev);
 
+#ifdef CONFIG_NET_LOOPBACK_DROP_ALL_PACKETS
+	return 0;
+#endif
+
```

Comment:
Does not make any difference here, but I would suggest that we try to use the `IS_ENABLED()` macro when possible. This way perhaps the use gets more widespread and becomes a habit.
```
if (IS_ENABLED(CONFIG_NET_LOOPBACK_DROP_ALL_PACKETS)) {
     return 0;
}
```